backend/src/openarchiefbeheer/destruction/tests/endpoints/test_co_reviews.py

```diff
@@ -0,0 +1,147 @@
+            f"{url}?destruction_list__uuid={destruction_list.uuid}"
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_list_filter_destruction_list__uuid(self):
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        DestructionListCoReviewFactory.create()
+        co_review_related = DestructionListCoReviewFactory.create(
+            destruction_list=destruction_list
+        )
+        user = UserFactory()
+
+        self.client.force_login(user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.get(
+            f"{url}?destruction_list__uuid={destruction_list.uuid}"
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(response.data[0]["pk"], co_review_related.pk)
+
+    def test_create_not_logged_in(self):
+        co_reviewer = DestructionListAssigneeFactory.create(role=ListRole.co_reviewer)
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(co_reviewer)
+
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_create_not_co_reviewer(self):
+        user = DestructionListAssigneeFactory.create()
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(user)
+
+        self.client.force_login(user.user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_create_not_assigned(self):
+        co_reviewer = DestructionListAssigneeFactory.create(role=ListRole.co_reviewer)
+        destruction_list = DestructionListFactory.create(
+            status=ListStatus.ready_to_review
+        )
+        destruction_list.assignees.add(co_reviewer)
+        user = UserFactory(post__can_co_review_destruction=True)
+
+        self.client.force_login(user)
+        url = reverse("api:destruction-list-co-reviews-list")
+        response = self.client.post(
+            url,
+            {
+                "destruction_list": destruction_list.uuid,
+                "list_feedback": "gh-497",
+            },
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
Maybe also check that the error is what we expect

Reply:
- [ ] Expand test